### ISSUE TITLE
Uncaught TypeError: Cannot read property 'profiler' of undefined

### ISSUE THREAD

**Issue by rkstar** [ultimatejs/tracker-react#16]
this is related to what i read in issue #11 -- 
i'm using [webpack:webpack](https://github.com/thereactivestack/meteor-webpack/tree/master/packages/webpack/) with meteor 1.3 and i get this error.

i tried adding tracker-react @ 1.0.1 thinking that maybe the issue that caused the problem with 1.3 modules would work with webpack modules.

any ideas here?  i tried using the `TrackerReactMixin` as a decorator too, but got this error:
`Uncaught TypeError: (0 , _ultimatejsTrackerReact.TrackerReactMixin) is not a function(anonymous function)`

**Comment by D1no** [comment]
That is odd. Please update to the latest version. It has been re-published as version 1.0.4. The mentioned issue should be unrelated. Please let me know if it works now.

If not, please post more of your code / reproduction. webpack:webpack received a major overhaul and also the way Meteor 1.3 packages can export variables (using the modules system) from module files, which is new and used by TrackerReact.

Not sure if @eXon has maybe **not implemented yet the loading of package-module export** into webpack.
https://github.com/ultimatejs/tracker-react/blob/master/package.js#L15

``` js
import TrackerReact from "meteor/ultimatejs:tracker-react"
```

but does not provide a global

``` js
// this alone will be undefined
console.log(TrackerReact) 
```

**Comment by rkstar** [comment]
i upgraded to 1.0.4 and i'm able to use the `TrackerReact()` function wrapper around `React.Component` now
